src/pages/Editor/containers/editContext.js

```diff
@@ -182,13 +182,13 @@ const EditContextProvider = props => {
 
   function convertToPlain(rtf) {
     rtf = rtf.replace(/\\par[d]?/g, "");
-    rtf = rtf.replace(/\{\*?\\[^{}]+}|[{}]|\\\n?[A-Za-z]+\n?(?:-?\d+)?[ ]?/g, "");
+    rtf = rtf.replace(/\{\*?\\[^{}]+}|\\\n?[A-Za-z]+\n?(?:-?\d+)?[ ]?/g, "");
     // rtf = rtf.replace(/\n/ig, " ");
-    rtf = rtf.replace(/\\/gi, "");
-    rtf = rtf.replace(/\*/gi, "");
+    // rtf = rtf.replace(/\\/gi, "");
```

Comment:
don't add these in comments remove them. if they are of no use.

Reply:
All right!

Reply:
I have committed the necessary changes. Please go through the same and merge the pull request.

Reply:
I have committed the necessary changes. Please go through the same and merge the pull request.